models/relation.rb

```diff
@@ -31,5 +31,10 @@ class Relation < Base
     # validates_numericality_of :lapse, allow_nil: true
     # validates_numericality_of :periodicity, greater_than_or_equal_to: 1
     # validates_inclusion_of :type, :in => %i(same_route sequence order minimum_day_lapse maximum_day_lapse shipment meetup maximum_duration_lapse vehicle_group_duration vehicle_group_duration_on_weeks vehicle_group_duration_on_months vehicle_trips)
+
+    def self.create(hash)
```

Comment:
Could we eventually use a coerce in grape model instead ?

Reply:
We don't need a coerce in the grape model actually. It is already corrected on the API level (and Grape does the string -> symbol conversion automatically) this is here to make sure that the internal model respects the model defined by the API.